tools/ld-chroma-decoder/monodecoder.cpp

```diff
@@ -28,6 +28,12 @@
 #include "decoderpool.h"
 #include "palcolour.h"
 
+MonoDecoder::MonoDecoder(const Comb::Configuration &combConfig)
```

Comment:
This doesn't look right - each Decoder subclass has its own Configuration subclass; it doesn't really make sense to pass the NTSC decoder's configuration to MonoDecoder. As MonoDecoder doesn't need anything that isn't in Decoder::Configuration then it just inherits it directly; you should pass a MonoDecoder::Configuration here, not a Comb::Configuration (and then the constructor doesn't need to do anything to convert it).

Reply:
At one point I passed it a palConfig.  All of the option setting in main.cpp is through palConfig and combConfig, but combConfig isn't an Encoder subclass so I don't know how to unify it all nicely.